packages/ui-components/src/components/utils.ts

```diff
@@ -12,3 +12,19 @@ export const getConfig = (): UIComponentsConfig => {
 
 	return DEFAULT_CONFIG;
 };
+
+export const getCollapsedElement = (elements: NodeListOf<ChildNode>): HTMLElement => {
```

Comment:
This function signature must be:

```suggestion
export const getCollapsedElement = (elements: NodeListOf<ChildNode>): HTMLElement | undefined => {
```

Correct?